internal/acctest/acctest.go

```diff
@@ -47,6 +47,11 @@ func RandomIP() string {
 	return fmt.Sprintf("%d.%d.%d.%d", rand.Intn(255), rand.Intn(255), rand.Intn(255), rand.Intn(255))
 }
 
+// RandomNumber Function to generate a random 12-digit number for cloud discovery Role ARN
+func RandomNumber() string {
+	return fmt.Sprintf("%d", rand.Intn(999999999999))
+}
+
```

Comment:
this function is not generic enough to be here, and be called RandomNumber :), maybe you can use this within the clouddiscovery package directly

Reply:
Can this be placed in model_discovery_config itself or should a separate acctest.go file created inside the cloud discovery folder ?